docs/getting-started/release-notes/v2_9_0.md

```diff
@@ -18,31 +18,91 @@ Zowe Version 2.9.0 contains the enhancements that are described in the following
 * `zowe.certificates.pem` is no longer needed when using keyrings. ([#448](https://github.com/zowe/zlux-server-framework/pull/448))
 
 #### Zowe Common C
-* `configmgr's` zOS module now has a `resolveSymbol` function which takes a string starting with and which can be used to resolve static and dynamic zos symbols. ([#378](https://github.com/zowe/zowe-common-c/pull/378))
+* `configmgr's` zOS module now has a `resolveSymbol` function which takes a string starting with `&` which can be used to resolve static and dynamic ZOS symbols. ([#378](https://github.com/zowe/zowe-common-c/pull/378))
 ### Zowe API Mediation Layer
 
+### Imperative CLI Framework
+
+- Added the function `IO.giveAccessOnlyToOwner` to restrict access to only the currently running user ID for security. ([#987](https://github.com/zowe/imperative/pull/987))
+
+- Enabled command arguments to change `{$Prefix}_EDITOR`. Updated `IDiffOptions`
+to include names for the files that are to be compared. Updated `IO.getDefaultTextEditor()` for different OS versions. Updated return value types for `CliUtils.readPrompt`. This prepares development of future commands in Zowe CLI. ([#967](https://github.com/zowe/imperative/pull/967))
+
+- Altered `TextUtils` behavior slightly to enable daemon color support without TTY. ([#977](https://github.com/zowe/imperative/pull/977))
+
 ### Zowe CLI
 
+- Updated daemon to use `tokio` library instead of the unmaintained `named_pipe` library for technical currency. ([#1710](https://github.com/zowe/zowe-cli/pull/1710))
+
+- Added the `zowe files copy dsclp` command to copy a data set from one LPAR to another, making data transfer between LPARs easier. ([#1098](https://github.com/zowe/zowe-cli/issues/1098))
+
+- Re-enabled color in the daemon client to differentiate text displayed in the terminal. ([#1379](https://github.com/zowe/zowe-cli/issues/1379))
+
 ### Zowe Explorer
 
+- Added option to save unique data set attributes as a template after allocation for future use. ([#1425](https://github.com/zowe/vscode-extension-for-zowe/issues/1425))
+
+- Added "Cancel Job" feature for job nodes in Jobs panel view. ([#2251](https://github.com/zowe/vscode-extension-for-zowe/issues/2251))
+
+- Enhanced ID generation for parent tree nodes to ensure uniqueness. ([#2325](https://github.com/zowe/vscode-extension-for-zowe/pull/2325))
+
+- Added support for custom credential manager extensions. ([#2212](https://github.com/zowe/vscode-extension-for-zowe/issues/2212))
 ## Bug fixes
```

Comment:
I don't think this will have an impact, but typically a newline is inserted after the content and before the next Markdown header.